go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/digitalocean/ceph_exporter
+
+go 1.15
+
+require (
+	github.com/beorn7/perks v0.0.0-20160804104726-4c0e84591b9a // indirect
+	github.com/ceph/go-ceph v0.0.0-20181217221554-e32f9f0f2e94
```

Comment:
Hmm, this seems like an older version of `go-ceph`, no? IIRC we had it at 0.5.0 on this branch previously.

Reply:
Ah! Good catch! This's definitely off.

Updated to the `0.6.0` from `go get`

```
$ GO111MODULE=on go mod tidy                                   12:00:31.667
go: finding module for package github.com/ceph/go-ceph/rados
go: downloading github.com/ceph/go-ceph v0.6.0
go: found github.com/ceph/go-ceph/rados in github.com/ceph/go-ceph v0.6.0
```

Reply:
:+1: